nbgrader/apps/dbapp.py

```diff
@@ -140,6 +140,7 @@ def start(self):
         with Gradebook(self.coursedir.db_url) as gb:
             with open(path, 'r') as fh:
                 reader = csv.DictReader(fh)
+                reader.fieldnames = self._preprocess_keys(reader.fieldnames)
```

Comment:
I just realized we need to do this also in `DbAssignmentImportApp`, as it suffers from the same issue---that trailing whitespace will cause it to not recognize keys. Can you add this same functionality there too?

Reply:
done!